weblab/static/js/upload.js

```diff
@@ -59,17 +56,30 @@ Upload.prototype = {
 
     $("form #entityversionfilestable").on('click', 'tr .action .delete-file', function() {
       var $tr = $(this).parents('tr');
+      return false;
```

Comment:
This function doesn't appear to be doing anything?

Reply:
If I make it call `toggleDelete($tr)` (fixing the error below) then I can't delete multiple files in one go, because clicking the delete icon performs a GET request. (Why it should do this when the handler returns false I'm not sure!) If I leave it as-is, nothing happens at all when I click the delete icon.